lsp-mode.el

```diff
@@ -3905,6 +3907,79 @@ A reference is highlighted only if it is visible in a window."
                       (push overlay buf-overlays)
                       (puthash (current-buffer) buf-overlays overlays))))))))))))
 
+(defvar lsp-semantic-highlighting-faces
```

Comment:
As I already mentioned the code should be changed so the suffix("cpp") is not part of the keys.